docs/sources/operations/storage/boltdb-shipper.md

```diff
@@ -113,6 +113,10 @@ We recommend running a Compactor since a single Ingester creates 96 files per da
 
 Example compactor configuration with GCS:
 
+#### Delete Permissions
+
+In the course of compacting index files, the compactor writes a new file and deletes the unoptimized ones. Therefore, ensure the compactor has appropriate permissions for deleting files, such as `s3:DeleteObject` for AWS S3.
```

Comment:
When compacting index files, the compactor writes a new file and deletes all unoptimized files. Ensure that the compactor has appropriate permissions for deleting files, for example, `s3:DeleteObject` permission for AWS S3.